frontend/src/components/ProfileComponent.vue

```diff
@@ -44,17 +46,26 @@
           <label for="formFile" class="form-label">Update Avatar <a href="#" class="fa fa-edit" :class="[inputsState ? 'd-none' : '']" @click="editInputs('avatar')"></a></label>
           <input class="form-control" type="file" id="formFile" ref="avatar" :onChange="pickImage" disabled>
         </div>
-        <div class="d-grid mb-4">
-          <button type="submit" class="btn btn-danger btn-lg"  @click="editInfo" :class="[buttonState ? 'd-none' : '']">
-            <span v-if="inputsState">Edit Info</span>
-            <span v-else>Cancel Edit</span>
-          </button>
-        </div>
-        <div class="d-grid mb-4">
-          <button type="submit" class="btn btn-primary btn-lg" :class="[inputsState ? 'd-none' : '']" @click="updateInfo" :disabled="buttonState">
-            <span v-if="!buttonState">Update Info</span>
-            <span v-else>Updating Profile please wait ...</span>
-          </button>
+        <div class="row">
+          <div class="col-md-6">
+            <div class="d-grid mb-4">
+              <a href="/my-activity-log" class="btn btn-warning btn-lg">Activity Log</a>
+            </div>
+          </div>
+          <div class="col-md-6">
+            <div class="d-grid mb-4">
+              <button type="submit" class="btn btn-danger btn-lg"  @click="editInfo" :class="[buttonState ? 'd-none' : '']">
```

Comment:
Does not have to be ternary if else condition is empty,
you can update other parts of your code to like this

```suggestion
              <button type="submit" class="btn btn-danger btn-lg"  @click="editInfo" :class="{'d-none': buttonState}">
```